### PR TITLE
It fails if ~/securedata/settings.json does not exist.

### PR DESCRIPTION
**Pull request by arvinddeshpande**
Fails if an empty string is entered when asked for the location.

Fails if directory already exists.

There's no need to store the location in config.json in the modules
directory. Since root can install it for all as a site-wide module.

e.g. instead of pip install securedata as pi or any other user, try sudo pip install secuedata and then run securedata config as a pi user and try to modify the location

### CODE REVIEW

File: src/securedata/securedata.py [hunk 1]
```diff
@@ -184,9 +194,7 @@ def setConfigItem(key=None, value=None):
     Updates the internal configuration file
     """
 
-    global configPath
-
-    if value == None:
+    if value == None or value == None:
```
Comment (tylerjwoodfin) [inline src/securedata/securedata.py:197]: Why is `value == None` being checked twice?

File: src/securedata/securedata.py [hunk 2]
```diff
@@ -288,4 +296,6 @@ def log(message=None, logName=None, level="info", filePath=None):
     print(f"SecureData is a library not intended to be directly run. See README.md.")
 
 if argv[-1] == 'config':
-    setConfigItem('path_securedata', input("Enter the full path of where you want to store all data:\n"))
\ No newline at end of file
+    readInput = input("Enter the full path of where you want to store all data:\n")
+    if len(readInput.strip()):
```
Comment (tylerjwoodfin) [inline src/securedata/securedata.py:300]: `setConfigItem` should already check this- it already has logic to show "No changes were made" and exit. I've made another commit to fix this part of it.

Comment (tylerjwoodfin) [reply src/securedata/securedata.py:300]: Please pull my latest changes and resolve the conflict, and I'll approve. The rest looks good 👍

Comment (arvinddeshpande) [reply src/securedata/securedata.py:301]: If user does not provide any input there's no need to call setItem

**Review by tylerjwoodfin — APPROVED** [review]

**Comment by tylerjwoodfin** [comment]
[Reverted- it was causing s](https://github.com/tylerjwoodfin/securedata/pull/3#issue-1097323314)

**Pull request merged**